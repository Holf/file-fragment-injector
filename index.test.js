const fs = require("fs");
const injectFileFragment = require("./index.js");

describe("File fragment injection", () => {
  it("should", () => {
    injectFileFragment(
      "./testAssets/sourceFile.html",
      "./testAssets/destFile.html"
    );

    const generatedFile = fs.readFileSync("./testAssets/destFile.html", "utf8");

    expect(generatedFile).toMatchSnapshot();
  });
});
