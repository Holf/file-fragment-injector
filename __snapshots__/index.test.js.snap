// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File fragment injection should 1`] = `
"<!doctype html>
<html lang="en">
  <head>
    <script>
      console.log("Hello from fragmentFile1.js");

    </script>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
  </head>
  <body>
    <div>This is some content from fragmentFile2.html</div>

  </body>
</html>
"
`;
